lib/chromic_pdf/template.ex

```diff
@@ -225,7 +257,8 @@ defmodule ChromicPDF.Template do
   def styles(opts \\ []), do: do_styles(opts)
 
   defp do_styles(opts) do
-    {width, height} = get_paper_size(opts)
+    landscape = Keyword.get(opts, :landscape, false)
```

Comment:
instead of doing this twice, how about we extract `landscape` in `get_paper_string(opts)`?